src/v0/destinations/active_campaign/transform.js

```diff
@@ -62,11 +62,11 @@ const syncContact = async (contactPayload, category, destination) => {
     feature: 'transformation',
   });
   if (res.success === false) {
-    errorHandler(res.response, 'Failed to create new contact');
+    errorHandler(res, 'Failed to create new contact');
```

Comment:
Doesn't this need update in `errorHandler` logic as well?

Reply:
If not how was it working previously?

Reply:
The `res` variable is of type `AxiosError` which contains `response`. And this `response` contains the details of the error in field `data`

Basically `res.response.data` contains the error response from `active_campaign`
Please see the screenshot for more clarification

![Screenshot 2023-12-12 at 13 14 28](https://github.com/rudderlabs/rudder-transformer/assets/18283111/5c3790d8-e103-4ae7-84c4-486c102f1903)

Reply:
For success, we are dealing normally & don't need to depend on `errorHandler` logic.

Reply:
ok